BGPViewerCore/Service/BGPHeService.cs

```diff
@@ -131,26 +131,32 @@ private IEnumerable<string[]> ExtractAsnsInfoFromPeerTable(IWebElement peerTable
             }
         }
 
-        private IEnumerable<string[]> ExtractPrefixAndAsnInfoFromNetInfoDivTable(string tableInnerHtml) 
+        private IEnumerable<string[]> ExtractPrefixAndAsnInfoFromNetInfoDivTable(IWebDriver driver) 
         {
-            var matches = Regex.Matches(tableInnerHtml, @"<a.*>.*<\/a>|<td>.*<\/td>");
-            var isMatchCorrect = matches.Count % 3 == 0; // Each table has to match 3 rows (asn, prefix and name)
```

Comment:
It's not clear why number 3 is being used here. Refactor it to a constant with a friendly name.